docs/compiling.rst

```diff
@@ -143,7 +143,7 @@ Your ``pyproject.toml`` file will likely look something like this:
 .. code-block:: toml
 
     [build-system]
-    requires = ["setuptools>=42", "wheel", "pybind11~=2.6.1"]
+    requires = ["setuptools>=42", "wheel", "pybind11>=2.6.1"]
```

Comment:
```suggestion
    requires = ["setuptools>=42", "pybind11>=2.6.1"]
```

(Honestly, this is a really old lower bound, given this is our docs, showing something recent also seems fine. Agree about [not having a cap](https://iscinumpy.dev/post/bound-version-constraints/)!)